crates/db_views/src/comment_view.rs

```diff
@@ -129,14 +131,15 @@ fn queries<'a>() -> Queries<
       Box::new(None::<bool>.into_sql::<sql_types::Nullable<sql_types::Bool>>())
     };
 
-    let is_saved_selection: Box<dyn BoxableExpression<_, Pg, SqlType = sql_types::Bool>> =
-      if saved_only {
-        Box::new(true.into_sql::<sql_types::Bool>())
-      } else if let Some(person_id) = my_person_id {
-        Box::new(is_saved(person_id))
-      } else {
-        Box::new(false.into_sql::<sql_types::Bool>())
-      };
+    let is_saved_selection: Box<
+      dyn BoxableExpression<_, Pg, SqlType = sql_types::Nullable<sql_types::Timestamptz>>,
+    > = if saved_only {
+      Box::new(None::<DateTime<Utc>>.into_sql::<sql_types::Nullable<sql_types::Timestamptz>>())
```

Comment:
I dont understand what this first if block is for.

Reply:
You're right, I don't think its necessary, I'll remove.

Reply:
I'm on senior disneyland trip so i'm not gonna look at that if statement right now, but the original if statements were there so the subquery in the filter is not done a 2nd time in the selection. Shouldn't be a big problem because the new joins in #4459 will make this unnecessary.